### PR TITLE
Skip chromatic deployment job for draft PRs

### PR DESCRIPTION
**Pull request by pczeglik-iohk**
# Checklist

- [ ] JIRA - \<link>
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

### CODE REVIEW

File: .github/workflows/core-chromatic.yml [hunk 1]
```diff
@@ -11,7 +12,15 @@ on:
       - main
 
 jobs:
+  fail_if_pull_request_is_draft:
+    if: github.event.pull_request.draft == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Fails for Draft PRs. Mark it ready for review to execute chromatic-deployment job
+      run: exit 1
+
```
Comment (przemyslaw-wlodek) [inline .github/workflows/core-chromatic.yml:21]: This one seems redundant 😄

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

File: packages/core/src/ui/components/SharedWallet/AllSet/AllSet.stories.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { AllSet } from './AllSet';
 import { ComponentProps } from 'react';
 
 const meta: Meta<typeof AllSet> = {
-  title: 'Shared Wallets/AllSet',
+  title: 'Trigger chromatic test run for draft',
```
Comment (DominikGuzei) [inline packages/core/src/ui/components/SharedWallet/AllSet/AllSet.stories.ts:7]: What has this story to do with chromatic draft runs?

Comment (pczeglik-iohk) [reply packages/core/src/ui/components/SharedWallet/AllSet/AllSet.stories.ts:7]: Please ignore this PR. I'm experimenting with CI :)

Comment (pczeglik-iohk) [reply packages/core/src/ui/components/SharedWallet/AllSet/AllSet.stories.ts:7]: This was a way to check triggering chromatic on draft vs pull requests

**Pull request closed without merging**